server/controllers/finance/reports/balance/index.js

```diff
@@ -58,6 +59,9 @@ function document(req, res, next) {
 
   getBalanceSummary(params.period_id, req.session.enterprise.currency_id, context.shouldPruneEmptyRows)
     .then(data => {
+      if (context.shouldHideTitleAccounts) {
+        data.accounts = data.accounts.filter(account => account.isTitleAccount === 0);
```

Comment:
This is the easiest way of doing it, but it is also the slowest.  Ideally, it would be nice to filter the title accounts in SQL to make things faster.